### PR TITLE
JS guide: improve user doc copywriting

### PR DESCRIPTION
**Pull request by bensonby**
connects #292
connects #251

### CODE REVIEW

**Comment by bensonby** [comment]
I don't know who to assign this to, perhaps to @cheungpat for the "lookup user by username" feature..

**Comment by cheungpat** [comment]
I think @rickmak has a better understanding of the user system.

File: content/js/users.md [hunk 1]
```diff
@@ -1,155 +1,295 @@
...
 });
 ```
 
-### Log out all other session (not yet implemented)
+Note: Changing the password of the current user will not trigger the callback
+registered through `skygear.onUserChanged`.
+
+[Not yet implemented] By setting `invalidateTokens` to `true`,
+you can invalidate the existing access token
+and get a new one so all other existing sessions become invalid.
```
Comment (rickmak) [inline content/js/users.md]: If we using JWT token store, all existing token will invalidate on changing password.

For other token, the invalidateTokens is not implemented.

**Comment by rickmak** [comment]
Other looks fine to me.

**Comment by bensonby** [comment]
The description for JWT token store has been added

**Pull request merged**